src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -784,23 +784,41 @@ private:
 #undef ON_EVENT
 
   static void parseETag(kj::StringPtr input, WebSession::ETag::Builder builder) {
+    auto maybePrintInvalidEtagWarning = [](kj::StringPtr input) -> void {
```

Comment:
Fair enough. I didn't want to add clutter to the namespace that `parseETag()` is in, but I am OK doing it if that's how you want it!